content/blog/accessing-hpe-data-fabric-s3-storage-from-spring-boot-s3-micro-service-deployed-in-k3s-cluster.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Accessing HPE Ezmeral Data Fabric S3 Storage from Spring Boot S3 Micro
+  Service deployed in K3S cluster
+date: 2021-09-13T08:13:40.734Z
+featuredBlog: true
```

Comment:
Hi Kiran,

Could you please set the option "FeaturedBlog" to "False". 

Thanks
Denis

Reply:
done